src/component/rocket/Rocket.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable arrow-body-style */
```

Comment:
For the sake of maintaining standard best practices, kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. :wink: